robot_smach_states/src/robot_smach_states/human_interaction/human_interaction.py

```diff
@@ -306,7 +306,7 @@ def __init__(self, robot, timeout=10):
 
 class AskYesNo(HearOptions):
     def __init__(self, robot, timeout=10):
-        HearOptions.__init__(self, robot, ['yes', 'no'], timeout)
+        HearOptions.__init__(self, robot, ['yes', 'no'], timeout, False)
```

Comment:
This is `look_at_standing_person`. Which is `True` by default. I think we should keep it that way. This was a change for the final only.